### DISCUSSION TITLE
Template Storage with dashes in names fail

### DISCUSSION THREAD

**Discussion by stevecopeza** [community-scripts/ProxmoxVE#777] [General]
I'm sure someone is aware of this, its on the off chance that its not the case that I share here.
I was battling to understand why I was getting inconsistent results using the scripts, with some failing right at the beginning. Long story short, if I selected storage with a dash "-" in the name, stuff would fail. I now try and not use dashes in share / storage naming conventions.

**Comment by michelroegl-brunner** [comment]
I can only say that this may be a problem on your side an has nothing to do with the scripts. I only use Storages with more than one dash in the name (e.g MSA2040-1-A or MSA2040-2-B, those are the real names btw.). Can you share the error message by chance if there is any?
